src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component, ReactNode } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends Component {
+  state: Readonly<{ key?: string }> = {
+    key: undefined,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({
+        key: event.key,
+      });
+    });
+
+    return (
+      document.removeEventListener('keyup', (event: KeyboardEvent) => {
+        this.setState({
```

Comment:
why do you return it? You should use here componentWillUnmount() to remove EventListener